src/main/java/com/epam/test_generator/services/UserService.java

```diff
@@ -17,15 +19,19 @@
 @Service
```

Comment:
Why is it not  transactional? There are operations with database and userDAO isn't transactional too.